bundle/scripts/build.js

```diff
@@ -2,72 +2,92 @@ const { execSync } = require("child_process");
 const path = require("path");
 const fs = require("fs");
 const { getSystemErrorMap } = require("util");
-const { argv } = require("process");
+const { argv, stdout, exit } = require("process");
+const { copyRecursiveSync, execIn, parts } = require('./utils');
 
-var steps = 2, i = 0;
+if(argv.includes("help")) {
+	console.log(`Fosscord build script help:
+Arguments:
+  clean			Cleans up previous builds
+  copyonly		Only copy source files, don't build (useful for updating assets)
+  verbose		Enable verbose logging
+  logerrors		Log build errors to console
+  pretty-errors		Pretty-print build errors
+  silent		No output to console or files.`);
+	exit(0);
+}
+
+let steps = 3, i = 0;
 if (argv.includes("clean")) steps++;
 if (argv.includes("copyonly")) steps--;
-const dirs = ["api", "util", "cdn", "gateway", "bundle"];
 
 const verbose = argv.includes("verbose") || argv.includes("v");
+const logerr = argv.includes("logerrors");
+const pretty = argv.includes("pretty-errors");
+const silent = argv.includes("silent");
 
-var copyRecursiveSync = function(src, dest) {
```

Comment:
Evaluate silent before verbose, logerrors and pretty-errors

Reply:
not needed, since silent just masks console.log etc